Code completion in completion7.xml for android:orientation="^ selecting horizontal:
@@ -5 +5
-     android:orientation="^
+     android:orientation="horizontal^
